### PR TITLE
Project 3 - Isabella Billgren

### PR DESCRIPTION
**Pull request by Bellsibub**
Completed all BLUE, RED and BLACK requirements.
(see readme for more info)

### CODE REVIEW

File: code/index.html
Comment (EmOlsz) [inline code/index.html:7]: Next time let's remember about `./` in path to file ;)

File: code/index.html
Comment (EmOlsz) [inline code/index.html:12]: Great job with BEM methodology!

File: code/index.html
Comment (EmOlsz) [inline code/index.html:18]: We shouldn't use `<br />` tags - more information on that in [Week 2 Master document](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/index.html
Comment (EmOlsz) [inline code/index.html:38]: If we decide to nest input inside label, then we don't need to add for - id pair. More details in Week 3 Master document

File: code/index.html
Comment (EmOlsz) [inline code/index.html:227]: Great accordion structure :)

File: code/script.js
Comment (EmOlsz) [inline code/script.js:8]: Very complex logic, I'm impressed!

File: code/styles/forms.css
Comment (EmOlsz) [inline code/styles/forms.css:83]: 👍

File: code/styles/forms.css
Comment (EmOlsz) [inline code/styles/forms.css:119]: Would be better to add separate class for `label`, let's remember about this next time!

File: code/styles/gridLayout.css
Comment (EmOlsz) [inline code/styles/gridLayout.css:7]: All commented code should be removed before final push - let's fix this

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Wow Isabella, congrats! ⭐️ Very nice, well done project. I'm impressed by your BEM implementation, custom input styles and JS logic. There are only super small things that needs to be fixed before approval, check them out :)
Moreover, feel free to analyse [Week 3 Master document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475)

**Comment by Bellsibub** [comment]
Requested changes have been pushed :)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**
